examples/forwarding-refs/customized-display-name.js

```diff
@@ -7,10 +7,10 @@ function logProps(Component) {
     return <LogProps {...props} forwardedRef={ref} />;
   }
 
-  // Give this component a more helpful display name in DevTools.
+  // Dê a este componente um nome mais visivelmente amigável no DevTools
   // e.g. "ForwardRef(logProps(MyComponent))"
   // highlight-range{1-2}
-  const name = Component.displayName || Component.name;
+  const nome = Component.displayName || Component.name;
```

Comment:
Somente valores de strings e comentários podem ser traduzidos.

```suggestion
  const name = Component.displayName || Component.name;
```